geomesa-spark/geomesa-spark-jts/src/main/scala/org/apache/spark/sql/jts/JTSTypes.scala

```diff
@@ -11,7 +11,7 @@ package org.apache.spark.sql.jts
 import com.vividsolutions.jts.geom._
 import org.apache.spark.sql._
 import org.apache.spark.sql.catalyst.InternalRow
-import org.apache.spark.sql.catalyst.expressions.GenericInternalRow
+import org.apache.spark.sql.catalyst.expressions.{GenericInternalRow, Literal}
```

Comment:
does this file need to change? seems like it could be rolled back

Reply:
Yep. Cruft from an edit that went away.